examples/Nexrad_S3_Demo.py

```diff
@@ -0,0 +1,117 @@
+"""
+Plotting AWS-hosted NEXRAD Level 2 Data
+=======================================================
+
+Access NEXRAD radar data via Amazon Web Services and plot with MetPy
+
+Accessing data remotely is a powerful tool for big data, such as NEXRAD radar data.
+By accessing it in the cloud, you can save time and space from downloading the data locally.
+
+"""
+
+from io import BytesIO
+
+import boto3
+import botocore
+from botocore.client import Config
+import matplotlib.pyplot as plt
+from metpy.io import Level2File
+from metpy.plots import add_timestamp, ctables
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+import numpy as np
+
+######################################################################
+# Access the data in the AWS cloud. Note that you'll need an AWS account (free at
+# https://aws.amazon.com/) and then set up security keys under the "My Security Credentials"
+# tab. Following that, you'll need to set up your credentials locally. Follow the instructions
+# here: http://boto3.readthedocs.io/en/latest/guide/quickstart.html#configuration. In this
+# example, we're plotting data from the Evansville, IN radar, which had convection within its
+# domain on 06/26/2019.
+#
+
+s3 = boto3.resource('s3', config=Config(signature_version=botocore.UNSIGNED,
+                                        user_agent_extra='Resource'))
+bucket = s3.Bucket('noaa-nexrad-level2')
+for obj in bucket.objects.filter(Prefix='2019/06/26/KVWX/KVWX20190626_221105_V06'):
+    print(obj.key)
+    data = obj.get()['Body'].read()
+    bytestream = BytesIO(obj.get()['Body'].read())
+
+    # Use MetPy to read the file
+    f = Level2File(bytestream)
```

Comment:
`data` is unused here. Does passing `obj.get()['Body']` not work here? I'm guessing `Level2File` needs `seek()`...and I wonder if boto does that.